src/App.jsx

```diff
@@ -53,45 +45,53 @@ function App() {
       description: "멕시칸 캐주얼 그릴",
       category: "기타",
     },
-  ];
+
+  ]);
+
+  const [category, setCategory] = useState("전체");
+  const [addModalOpen, setAddModalOpen] = useState(false);
+  const [detailModalOpen, setDetailModalOpen] = useState(false);
```

Comment:
```suggestion
  const [isOpenAddModal, setIsOpenAddModal] = useState(false);
  const [isOpenDetailModal, setIsOpenDetailModal] = useState(false);
```
보통 boolean 값을 다루는 변수명은 `is`, `has`, `should` 같은 접두사를 사용하는 것 같긴해요!🙃

Reply:
아하 그렇군요 저는 왜 붙어있나 했는데 그게 boolean임을 알려주는 코드였다니 ... !!